.github/workflows/main.yaml

```diff
@@ -23,7 +23,9 @@ jobs:
   ubuntu-20-04:
     # The type of runner that the job will run on
     runs-on: ubuntu-20.04
-    timeout-minutes: 20
+    # Do not run when the following string is used in the commit message.
```

Comment:
this was an experiment in brink... and it does not work for pull requests.

Reply:
Thanks, have removed those.